core/src/solvers/joint_interpolation.cpp

```diff
@@ -140,12 +140,12 @@ bool JointInterpolationPlanner::plan(const planning_scene::PlanningSceneConstPtr
 		// TODO(v4hn): planners need a way to add feedback to failing plans
 		// in case of an invalid solution feedback should include unwanted collisions or violated constraints
 		RCLCPP_WARN(LOGGER, "IK failed for pose target");
-		return false;
```

Comment:
:point_up: I guess this todo can be removed? Once you're done with this PR? Offering the option to return an error string counts as feedback I guess